### PR TITLE
similar to PR #3922 but for splitinj

### PR DESCRIPTION
**Pull request by LorenzoPiccari**
This PR also stops `pycbc_make_bank_verifier_workflow` from assuming the fixed list of ifos H1, L1, V1 .

### CODE REVIEW

File: bin/workflows/pycbc_make_bank_verifier_workflow [hunk 1]
```diff
@@ -176,7 +176,7 @@ inp_bank = workflow.cp.get('workflow', 'input-bank')
 inp_bank = wf.resolve_url_to_file(inp_bank)
 inp_bank.tags = []
 inp_bank.description='TEMPLATEBANK'
-inp_bank.ifo_list=(['H1','L1','V1'])
+inp_bank.ifo_list = (workflow.ifos)
```
Comment (titodalcanton) [inline bin/workflows/pycbc_make_bank_verifier_workflow:179]:
```suggest
inp_bank.ifo_list = workflow.ifos
```

File: bin/pycbc_split_inspinj [hunk 2]
```diff
@@ -44,7 +44,17 @@ if args.num_splits:
...
+# The : split is needed for columns like `process:process_id`,
+# which must be listed as `process:process_id` in `lsctables.New()`,
+# but are listed as just `process_id` in the `columnnames` attribute
+used_columns = []
+for col in allinjs.validcolumns:
+    att = col.split(':')[-1]
+    if att in allinjs.columnnames:
+        used_columns.append(col)
+
+new_inj_tables = [lsctables.New(tabletype, columns=used_columns) \
```
Comment (titodalcanton) [inline bin/pycbc_split_inspinj:57]:
```suggestion
new_inj_tables = [lsctables.New(tabletype, columns=used_columns)
```

**Review by titodalcanton — APPROVED** [review]

**Comment by pannarale** [comment]
I'll also ping @ahnitz, since this is also appearing in one of his latest PRs.

**Comment by titodalcanton** [comment]
It looks like the failure is an unrelated issue which is being resolved by @ahnitz.

**Comment by pannarale** [comment]
https://github.com/gwastro/pycbc/issues/3964 ?

**Comment by titodalcanton** [comment]
More like https://github.com/JohannesBuchner/UltraNest/issues/56.

**Pull request merged**